GameFiles/IW4x/userraw/scripts/_integration.gsc

```diff
@@ -653,7 +668,37 @@ TeamSwitchImpl()
     return self.name + "^7 switched to " + self.team;
 }
 
-HideImpl()
+LockControlsImpl( target )
+{
+    if ( !IsAlive( self ) )
+    {
+        return self.name + "^7 is not alive";
+    }
+    
+    target freezeControlsWrapper(true);
+    target God(true);
+    target Hide();
+    
+    target thread maps\mp\gametypes\_hud_message::oldNotifyMessage( "ALERT", "You have been frozen!", "compass_waypoint_target", ( 1, 0, 0 ), "ui_mp_nukebomb_timer", 7.5 );
```

Comment:
Let's either not use this, or call the the existing hud function. This code is not compatible with other cods and I'd like to use as little as possible